app/components/new-signup/checkbox.hbs

```diff
@@ -0,0 +1,31 @@
+<div class='user-details'>
+  <h2 class='user-details__heading' data-test-signup-form-label>
+    {{this.label}}
+  </h2>
+
+  <div class='role-details__field'>
+    {{#each this.checkboxData as |data|}}
+      <label class='checkbox-label'>
+        <input
+          type='checkbox'
+          name={{data.name}}
+          checked={{false}}

```

Comment:
checked will be false everytime?

Reply:
No when it is unchecked i set it false
because when it checked it will be true
see code - https://github.com/Real-Dev-Squad/website-my/pull/435/files#diff-c2db5896e347104e3e95f3c01e252ea4de1f798b9eff1995a531418bf83fcf04R34